checkov/sca_package_2/output.py

```diff
@@ -372,11 +393,14 @@ def create_package_overview_table_part(
             package_alias = get_package_alias(package_name, package_version)
             is_root = package_alias == root_package_alias
             is_public_overview = "(Public)" if cve['is_private_fix'] is False else ""
+            reachability = "Package Used" if ("IsUsed" in cve.get('reachability_risk_factors', {}) and
+                                              cve.get("reachability_risk_factors").get("IsUsed")) else ""
```

Comment:
You can chain the `.get` so that you don't have to repeat the first part of the condition

```suggestion
            reachability = "Package Used" if cve.get('reachability_risk_factors', {}).get("IsUsed", False) else ""
```